packages/dd-trace/src/appsec/index.js

```diff
@@ -202,7 +202,6 @@ function onRequestCookieParser ({ req, res, abortController, cookies }) {
 
 function onResponseBody ({ req, body }) {
   if (!body || typeof body !== 'object') return
-  if (!apiSecuritySampler.isSampled(req)) return
```

Comment:
if we remove this here, we are going to call to the waf always with the response body, it is an overhead that we try to minimize, I think that the thing that we want to do here is:

```
if (!apiSecuritySampler.sampleRequest(req, res)) return;
```